packages/react/src/Row/Row.test.tsx

```diff
@@ -64,6 +65,16 @@ describe('Row', () => {
     expect(component).toBeInTheDocument()
   })
 
+  rowTags.forEach((tag) => {
```

Comment:
We can delete the `'renders with an article tag'` and `'renders with a section tag'` tests